data/sql/updates/pending_db_world/Kaskala.sql

```diff
@@ -0,0 +1,423 @@
+(22, 7, 2576200, 9, 0, 31, 1, 3, 25762, 111352, 0, 0, 0, '', 'Vrykul Landing Ship - GUID 7'),
+(22, 8, 2576200, 9, 0, 31, 1, 3, 25762, 111274, 0, 0, 0, '', 'Vrykul Landing Ship - GUID 8'),
+(22, 9, 2576200, 9, 0, 31, 1, 3, 25762, 111276, 0, 0, 0, '', 'Vrykul Landing Ship - GUID 9'),
+(22, 10, 2576200, 9, 0, 31, 1, 3, 25762, 111358, 0, 0, 0, '', 'Vrykul Landing Ship - GUID 10'),
+(22, 11, 2576200, 9, 0, 31, 1, 3, 25762, 111271, 0, 0, 0, '', 'Vrykul Landing Ship - GUID 11'),
+(22, 12, 2576200, 9, 0, 31, 1, 3, 25762, 111354, 0, 0, 0, '', 'Vrykul Landing Ship - GUID 12'),
+(22, 13, 2576200, 9, 0, 31, 1, 3, 25762, 111300, 0, 0, 0, '', 'Vrykul Landing Ship - GUID 13'),
+(22, 14, 2576200, 9, 0, 31, 1, 3, 25762, 111356, 0, 0, 0, '', 'Vrykul Landing Ship - GUID 14');
+
+-- Set State Emote (Kvaldir Raider)
+UPDATE `creature_template_addon` SET `emote` = 375 WHERE (`entry` = 25760);
+
+-- Set SmartAI (Kvaldir Raider)
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 25760;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 25760);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(25760, 0, 0, 1, 38, 0, 100, 0, 0, 1, 0, 0, 0, 0, 80, 2576000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kvaldir Raider - On Data Set 0 1 - Run Script'),
+(25760, 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 37, 120000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kvaldir Raider - On Data Set 0 1 - Kill Self'),
+(25760, 0, 2, 3, 34, 0, 100, 0, 0, 1, 0, 0, 0, 0, 101, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kvaldir Raider - On Reached Point 1 - Set Home Position'),
+(25760, 0, 3, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 49, 0, 0, 0, 0, 0, 0, 9, 25764, 0, 50, 0, 0, 0, 0, 0, 'Kvaldir Raider - On Reached Point 1 - Start Attacking');
+
+-- Set Action List (Kvaldir Raider)
+DELETE FROM `smart_scripts` WHERE (`source_type` = 9 AND `entryorguid` = 2576000);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2576000, 9, 0, 0, 0, 0, 100, 0, 3000, 3000, 0, 0, 0, 0, 203, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kvaldir Raider - Actionlist - Exit vehicle'),
```

Comment:
I do see the Raiders dismount the vehicle at the same position and at the same time. It's hard to tell from the video, but they seem to dismount at the same location. but are staggered in time. Would changing the 3000, 3000 here to X, Y be an easy change to emulate this?